BigBaseV2/src/memory/pattern.cpp

```diff
@@ -72,14 +76,14 @@ namespace memory
 			}
 			else
 			{
-				m_bytes.push_back(std::nullopt);
+				m_bytes.push_back({});
 			}
 		}
 	}
 
 	pattern::pattern(const void *bytes, std::string_view mask)
 	{
-		for (std::size_t i = 0; i < mask.size(); ++i)
+		for (std::size_t i{}; i != mask.size(); ++i)
```

Comment:
Does the compiler optimise this mask.size() call or does it call .size() each iteration?

Reply:
Should do so, commiting changes now.

Reply:
Committed requested changes.